src/main/views/common/correspondence-address-manual/capture-correspondence-address-manual.njk

```diff
@@ -105,10 +105,12 @@
               } if errors.addressPostcode
           }) }}
     </div>
+     {% if typeOfBusiness === "LIMITED_COMPANY" or typeOfBusiness === "LIMITED_PARTNERSHIP" or typeOfBusiness === "LIMITED_LIABILITY_PARTNERSHIP" or typeOfBusiness === "PARTNERSHIP" or typeOfBusiness === "SOLE_TRADER" or typeOfBusiness === "CORPORATE_BODY" %}
```

Comment:
Whats the else of this if?

Reply:
We do not need else here I am doing if end here. So if not this then it will not print anything